http://www.vdr-portal.de/board/thread.php?postid=525780#post525780
fix problems on search function with browser Opera

Joerg Bornkessel <hd_brummy@gentoo.org> 19 Oct 2006
diff -Naur vdradmin-am-3.4.7.orig/template/default/navigation.html vdradmin-am-3.4.7/template/default/navigation.html
--- vdradmin-am-3.4.7.orig/template/default/navigation.html	2006-10-19 16:11:26.164534802 +0200
+++ vdradmin-am-3.4.7/template/default/navigation.html	2006-10-19 16:12:23.532120051 +0200
@@ -66,7 +66,7 @@
 
 		<form action="vdradmin.pl" method="get" name="FormName" target="main" class="search">
 			<input type="text" name="search" /><br />
-			<input type="submit" name="submit" value="<?% gettext('Search') %?>" /
+			<input type="submit" name="submit" value="<?% gettext('Search') %?>" />
 			<input type="hidden" name="aktion" value="prog_summary" />
 		</form>
 	</div>
